apps/src/p5lab/spritelab/blocks.js

```diff
@@ -609,7 +609,6 @@ export default {
           block.setHSV(136, 0.84, 0.8);
           block.parameterNames_ = [i18n.thisSprite()];
           block.parameterTypes_ = [Blockly.BlockValueType.SPRITE];
-          block.setUserVisible(false);
```

Comment:
I'm not 100% sure why, but this is causing behaviors to break entirely - I thought it would be a purely UI change but it doesn't appear to be. I will dig into this more, but I think we should go ahead with this PR without this change since it'll still fix the majority of the lag

Reply:
fyi @epeach, but lmk if you have any concerns about this!